oauthapplication/client_test.go

```diff
@@ -36,9 +36,9 @@ func TestOAuthApplicationClientCreate(t *testing.T) {
 
 	params := &CreateParams{
 		Name:        name,
-		CallbackURL: callbackURL,
-		Scopes:      scopes,
-		Public:      public,
+		CallbackURL: clerk.String(callbackURL),
+		Scopes:      clerk.String(scopes),
```

Comment:
🙃 What if we forced the scopes to be a `[]string`, since it's a breaking change, and then joined them when creating the params for the Backend API?

Reply:
Yeah definitely agree here! It's on my list for the next major API version